be/src/connector/connector.h

```diff
@@ -127,7 +127,7 @@ class DataSourceProvider {
 public:
     static constexpr int64_t MIN_DATA_SOURCE_MEM_BYTES = 16 * 1024 * 1024;  // 16MB
     static constexpr int64_t MAX_DATA_SOURCE_MEM_BYTES = 256 * 1024 * 1024; // 256MB
-    static constexpr int64_t PER_FIELD_MEM_BYTES = 4 * 1024 * 1024;         // 4MB
+    static constexpr int64_t PER_FIELD_MEM_BYTES = 1 * 1024 * 1024;         // 1MB
```

Comment:
Why changed to 1M?

Reply:
sigh. really no specific reason. I just feel  4M maybe is too large.